pkg/langruntime/langruntime.go

```diff
@@ -223,10 +223,13 @@ func (l *Langruntimes) GetBuildContainer(runtime, depsChecksum string, env []v1.
 		command = appendToCommand(command,
 			"npm config set "+scope+"registry "+registry,
 			"npm install --production --prefix="+installVolume.MountPath)
+	case strings.Contains(runtime, "csharpx"):
+		command = appendToCommand(command,
+			"export HOME="+installVolume.MountPath,
+			"dotnet restore --packages "+installVolume.MountPath+"/packages > /dev/termination-log 2>&1")
```

Comment:
I had many problems regarding using a non-root user inside Kubernetes on `microsoft/aspnetcore-build:2.0` image. We you able to make this base image to work?
This image is using a user with `uid=1000` (instead of the default `root` user). Since it is the first time this `1000` user is using `dotnet restore` command, it will create a `.nuget` folder on the home directory of this user. The problem is that the home folder for this user is `/` - which it doesn't have any permissions to create directories there.
To overcome this barrier, I previously create the directory and move it. Check the Dockerfile:
```Dockerfile
FROM microsoft/aspnetcore-build:2.0.0

WORKDIR /app

COPY kubeless.csproj .

RUN dotnet restore

RUN mv /root/.dotnet /root/.nuget /

RUN chown 1000 /.dotnet/ /.nuget/
```